lib/CMakeLists.txt

```diff
@@ -13,487 +12,91 @@
-)
-
-if(UFE_FOUND)
-    list(APPEND mayaUsdUfe_headers
-        ufe/Global.h
-        ufe/ProxyShapeHandler.h
-        ufe/ProxyShapeHierarchy.h
-        ufe/ProxyShapeHierarchyHandler.h
-        ufe/StagesSubject.h
-        ufe/UsdHierarchy.h
-        ufe/UsdHierarchyHandler.h
-        ufe/UsdRootChildHierarchy.h
-        ufe/UsdRotatePivotTranslateUndoableCommand.h
-        ufe/UsdRotateUndoableCommand.h
-        ufe/UsdScaleUndoableCommand.h
-        ufe/UsdSceneItem.h
-        ufe/UsdSceneItemOps.h
-        ufe/UsdSceneItemOpsHandler.h
-        ufe/UsdStageMap.h
-        ufe/UsdTransform3d.h
-        ufe/UsdTransform3dHandler.h
-        ufe/UsdTranslateUndoableCommand.h
-        ufe/UsdUndoDeleteCommand.h
-        ufe/UsdUndoDuplicateCommand.h
-        ufe/UsdUndoRenameCommand.h
-        ufe/Utils.h
-		ufe/UfeVersionCompat.h
-    )
-
-    if(CMAKE_UFE_V2_FEATURES_AVAILABLE)
-        list(APPEND mayaUsdUfe_headers
-            ufe/UsdAttribute.h
-            ufe/UsdAttributes.h
-            ufe/UsdAttributesHandler.h
-            ufe/UsdObject3d.h
-            ufe/UsdObject3dHandler.h
-            ufe/UsdUndoCreateGroupCommand.h
-        )
-    endif()
-endif()
-
-list(APPEND mayaUsdPxrUsdMayaGL_headers
-    render/pxrUsdMayaGL/batchRenderer.h
-    render/pxrUsdMayaGL/debugCodes.h
-    render/pxrUsdMayaGL/hdImagingShapeDrawOverride.h
-    render/pxrUsdMayaGL/hdImagingShapeUI.h
-    render/pxrUsdMayaGL/hdRenderer.h
-    render/pxrUsdMayaGL/instancerImager.h
-    render/pxrUsdMayaGL/instancerShapeAdapter.h
-    render/pxrUsdMayaGL/proxyDrawOverride.h
-    render/pxrUsdMayaGL/proxyShapeUI.h
-    render/pxrUsdMayaGL/renderParams.h
-    render/pxrUsdMayaGL/sceneDelegate.h
-    render/pxrUsdMayaGL/shapeAdapter.h
-    render/pxrUsdMayaGL/softSelectHelper.h
-    render/pxrUsdMayaGL/usdProxyShapeAdapter.h
-    render/pxrUsdMayaGL/userData.h
-)
-
-list(APPEND mayaUsdPxVP20_headers
-    render/px_vp20/glslProgram.h
-    render/px_vp20/utils.h
-    render/px_vp20/utils_legacy.h
-)
-
-add_library(${LIBRARY_NAME} SHARED
-    ${mayaUsd_src}
-)
-
-set(compile_definitions_list
-    MAYAUSD_MACROS_EXPORT
-    MAYAUSD_CORE_EXPORT
-    MFB_PACKAGE_NAME="${LIBRARY_NAME}"
-    MFB_ALT_PACKAGE_NAME="${LIBRARY_NAME}"
-    MFB_PACKAGE_MODULE=mayaUsd
-)
-
-if(UFE_FOUND)
-    list(APPEND compile_definitions_list WANT_UFE_BUILD=1)
-endif()
-if(IS_MACOSX)
-    list(APPEND compile_definitions_list OSMac_)
-endif()
-
-target_compile_definitions(${LIBRARY_NAME} PRIVATE
-    ${compile_definitions_list}
+# -----------------------------------------------------------------------------
+# compile definitions
+# -----------------------------------------------------------------------------
+target_compile_definitions(${PROJECT_NAME} 
+    PRIVATE
+        MAYAUSD_MACROS_EXPORT
+        MAYAUSD_CORE_EXPORT
+        MFB_PACKAGE_NAME="${PROJECT_NAME}"
+        MFB_ALT_PACKAGE_NAME="${PROJECT_NAME}"
+        MFB_PACKAGE_MODULE=mayaUsd
```

Comment:
`${PROJECT_NAME}` here too instead of the literal `mayaUsd`?

Reply:
Good point. Will fix it in the next commit.

Reply:
Please see e25fa484